scipy/stats/_distribution_infrastructure.py

```diff
@@ -366,44 +368,6 @@ def contains(self, item, parameter_values=None):
         in_right = item <= b if right_inclusive else item < b
         return in_left & in_right
 
-
-class _RealDomain(_SimpleDomain):
```

Comment:
This is mostly just *moved* to below, and I moved the `draw` method to `_SimpleDomain`.